Code/Editor/EditorPreferencesPageViewportGeneral.h

```diff
@@ -44,6 +43,7 @@ class CEditorPreferencesPage_ViewportGeneral
         float m_defaultFOV;
         float m_defaultAspectRatio;
         bool m_enableContextMenu;
+        bool m_stickySelect;
```

Comment:
ultra nit: m_enableStickySelect perhaps?

Reply:
Hmm I'm not a fan of this as `enable` is a verb and makes much more sense as in the context of a function name as opposed to a variable. We could have it be called `m_stickySelectEnabled` (I'm not that fussed but I can update it (and the other name while I'm here if you think that's a good idea))

Reply:
Now renamed :+1: